crates/node_binding/binding.d.ts

```diff
@@ -1729,7 +1740,7 @@ export interface RawOptions {
 export interface RawOutputOptions {
   path: string
   pathinfo: boolean | "verbose"
-  clean: boolean
+  clean: boolean | JsCleanFilter
```

Comment:
cannot convert from
```javascript
output: {
  clean: {
    keep: "<str>",
  }
}
```
Complaining expecting a `boolean` but got an `object`.

Reply:
I guess you might need to modify packages/rspack/src/config/zod.ts.